src/main/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibrator.java

```diff
@@ -93,14 +93,14 @@
  *
  * <h4>Recalibrating SNPs in exome data</h4>
  * <pre>
- * ./gatk-launch VariantRecalibrator \
- *   -R reference.fasta \
- *   -V input.vcf \
- *   --resource hapmap,known=false,training=true,truth=true,prior=15.0 hapmap_3.3.b37.sites.vcf \
- *   --resource omni,known=false,training=true,truth=false,prior=12.0 1000G_omni2.5.b37.sites.vcf \
- *   --resource 1000G,known=false,training=true,truth=false,prior=10.0 1000G_phase1.snps.high_confidence.vcf \
- *   --resource dbsnp,known=true,training=false,truth=false,prior=2.0 dbsnp_135.b37.vcf \
- *   -an QD -an MQ -an MQRankSum -an ReadPosRankSum -an FS -an SOR -an InbreedingCoeff \
+ * gatk VariantRecalibrator \
+ *   -R Homo_sapiens_assembly38.fasta \
+ *   -V input.vcf.gz \
+ *   --resource hapmap,known=false,training=true,truth=true,prior=15.0 hapmap_3.3.hg38.sites.vcf.gz \
+ *   --resource omni,known=false,training=true,truth=false,prior=12.0 1000G_omni2.5.hg38.sites.vcf.gz \
+ *   --resource 1000G,known=false,training=true,truth=false,prior=10.0 1000G_phase1.snps.high_confidence.hg38.vcf.gz \
+ *   --resource dbsnp,known=true,training=false,truth=false,prior=2.0 Homo_sapiens_assembly38.dbsnp138.vcf.gz \
+ *   -an QD -an MQ -an MQRankSum -an ReadPosRankSum -an FS -an SOR \
  *   -mode SNP \
  *   --recalFile output.recal \
```

Comment:
If i'm not mistaken, it looks like there isn't actually an argument for --recalFile on this class. Should I remove mention of it from the docs? Is there supposed to be one?

Reply:
Sorry, that's GATK3 style.  That should be --output.  I prefer to leave the extension as .recal (it gets parsed as a VCF, but it doesn't have all the data you'd expect), thought @sooheelee might have thoughts.